cmd/cli/plugin/cluster/go.mod

```diff
@@ -0,0 +1,245 @@
+module github.com/vmware-tanzu/tanzu-framework/plugin/cluster
+
+go 1.17
+
+replace (
+	github.com/vmware-tanzu/tanzu-framework => ../../../../
```

Comment:
I believe we will be able to remove this dependency after https://github.com/vmware-tanzu/tanzu-framework/pull/3475 gets in.